test/test_summarizer_translation.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from haystack.schema import Document
+from haystack.pipelines import TranslationWrapperPipeline, SearchSummarizationPipeline
+from haystack.nodes import DensePassageRetriever, EmbeddingRetriever
+
+SPLIT_DOCS = [
```

Comment:
Minuscule detail: you can import this constant from the other suite. I generally wouldn't import from a suite to the other, but these two are very tied together and this import might highlight that a bit.

Reply:
done